package.json

```diff
@@ -18,7 +18,8 @@
     "react-toastify": "^10.0.5",
     "styled-components": "^6.0.2",
     "uid": "^2.0.2",
-    "use-local-storage-state": "^19.3.1"
+    "use-local-storage": "^3.0.0",
```

Comment:
I assume you installed this library by accident;

To remove it, write `npm uninstall use-local-storage` to your terminal